### PR TITLE
build: add unidock_tools dependency in toml

### PR DESCRIPTION
**Pull request by dp-yuanyn**

### CODE REVIEW

File: unidock_tools/Dockerfile [hunk 1]
```diff
@@ -27,5 +27,5 @@ RUN pip install tqdm
 
 COPY . /opt/unidock_tools
 RUN cd /opt/unidock_tools && \
-    pip install . && \
+    pip install . --no-deps && \
```
Comment (caic99) [inline unidock_tools/Dockerfile:30]: Why no-deps here?

File: unidock_tools/README.md [hunk 2]
```diff
@@ -14,15 +14,15 @@ Main features are:
...
-- RDKit
-- networkx
+- Python >= 3.8
 - MGLTools, if you want to use AD4 scoring function in Uni-Dock
 - mcdock dependencies, see [MCDock Installation](./MCDOCK.md#installation)
 
 ## Install
 
-- ```pip install .```
+In Uni-Dock repo, unidock_tools subdictionary, run ```pip install .```
```
Comment (caic99) [inline unidock_tools/README.md:23]:
```suggestion
Clone Uni-Dock repo, and under `unidock_tools` subdirectory run ```pip install .```
```

File: unidock_tools/Dockerfile (hunk 1 above)
Comment (dp-yuanyn) [reply unidock_tools/Dockerfile:30]: All dependencies have been installed by conda in previous build stages, there is no need to install them again.

Comment (caic99) [reply unidock_tools/Dockerfile:30]: I would suggest use dependencies solved in pip, but not installing them ahead in conda manually

**Review by caic99 — APPROVED** [review]

File: unidock_tools/Dockerfile (hunk 1 above)
Comment (dp-yuanyn) [reply unidock_tools/Dockerfile]: > I would suggest use dependencies solved in pip, but not installing them ahead in conda manually

Done. Dependencies will be installed by pip if they are in pypi.

**Pull request merged**